src/services/web3/transactions/concerns/contract.concern.ts

```diff
@@ -54,7 +66,9 @@ export class ContractConcern extends TransactionConcern {
       ]);
 
       trackGoal(Goals.ContractTransactionSubmitted);
-      return await contractWithSigner[action](...params, txOptions);
+
+      const result = await contractWithSigner[action](...params, txOptions);
+      return result;
```

Comment:
Why break this onto two lines?

Reply:
Probably forgot to undo after debugging.